### ISSUE TITLE
cannot decrement begin list iterator at "Peer is sending messages faster than allowed"

### ISSUE THREAD

**Issue by Fruchtzwerg94** [SwabianCoin/SwabianCoin#5]
STL verification failed at:
```
history_list.clear();
LOG(INFO) << "Peer is sending messages faster than allowed: " << history_list.front() << " " << history_list.back() << " " << history_list.size();
```
> cannot decrement begin list iterator

since `history_list` is empty and `history_list.back()` is called.

In: [PeersMonitor.cpp line 48](https://github.com/SwabianCoin/SwabianCoin/blob/0bd3723908e2ec4b7426d29a01b9431638455104/src/scn/BlockchainManager/PeersMonitor.cpp#L48)

**Comment by SwabianCoin** [comment]
Thanks for the hint. Yes, that piece of code does not make any sense. I will fix it in the next days.

**Comment by SwabianCoin** [comment]
fixed with Release 20.02.03
